package/yast2-drbd.changes

```diff
@@ -1,3 +1,15 @@
+-------------------------------------------------------------------
+Tue Nov 21 08:33:05 UTC 2017 - nwang@suse.com
+
+- bsc#1069132, fix error message when write resource conf to disk.
+- Version 3.3.1
```

Comment:
If you release both fixes at once you can merge the changes and release it as a single version update. In this case 3.3.0 will never reach the customers anyway.

Reply:
Got it.